src/MainWindow_TEST.cc

```diff
@@ -342,7 +342,7 @@ TEST(MainWindowTest,
 
   // Access window after it's open
   bool closed{false};
-  QTimer::singleShot(300, [&closed]
+  QTimer::singleShot(300, App(), [&closed]
   {
     auto win = App()->findChild<MainWindow *>();
```

Comment:
should this be:
```suggestion
  auto *app = App();
  QTimer::singleShot(300, app, [&closed]
  {
    auto *win = app->findChild<MainWindow *>();
```
?

Reply:
Yes, I went ahead an addressed the rest of the places where we do this.